src/core/IPosisBundle.d.ts

```diff
@@ -0,0 +1,9 @@
+// Bundle for programs that are logically grouped
+interface IPosisBundle {
+	install(registry: IPosisProcessRegistry): void;
+
+	// image name of root process in the bundle, if any
+	rootImageName?: string;
+	// default start context for root process, if any
+	defaultStartContext?: any;
```

Comment:
Yeah, I'm not a huge fan of this either. If you're looking to initialize your variables (just for your root process, too) then it seems fine to have to check "isInitialized" or something. Part of your root process's job can be to set up the environment, and making that explicit is good.

I also just don't see how I ever want to start a bundle without knowing the starting parameters.